lib/elements/satin_column.py

```diff
@@ -1479,21 +1499,31 @@ def to_stitch_groups(self, last_patch=None):
             patch += self.do_center_walk()
```

Comment:
Oh.  I forgot this `patch` was here too!  Since you're here, could you change this variable to `stitch_group` too?